website/gatsby-config.js

```diff
@@ -45,7 +45,20 @@ module.exports = {
     {
       resolve: 'gatsby-plugin-google-fonts',
       options: {
-        fonts: ['roboto mono'],
+        fonts: [
+          'roboto mono',
+          // https://design.google/library/spectral-new-screen-first-typeface/
+          // 'Spectral:400,700',
+          // https://design.google/library/choosing-web-fonts-beginners-guide/
+          // 'Libre+Baskerville:400,400i,700',
+          'Raleway:400,400i,700,700i',
+        ],
+      },
+    },
+    {
+      resolve: `gatsby-plugin-favicon`,
```

Comment:
This plugin generates the favicons (using a webpack plugin) based on the svg logo

Reply:
I think we might actually use that for solving #257